examples/i2s-controller-demo/Cargo.toml

```diff
@@ -12,6 +12,7 @@ cortex-m = "0.6.2"
 cortex-m-rtic = "0.5.3"
 rtt-target = {version = "0.2.0", features = ["cortex-m"] }
 nrf52840-hal = { features = ["rt"], path = "../../nrf52840-hal" }
+aligned = "0.3.4"
```

Comment:
Let's not add a dependency on aligned; it pulls in an unreasonable amount of transitive deps for something that is really just `#[repr(align(4))]`.

Reply:
Great idea!